build.sbt

```
@@ -200,6 +202,79 @@ lazy val sdkSpringTestKit = project
       "java.lang"))
   .settings(Dependencies.sdkSpringTestKit)
 
+lazy val sdkSpringBootStarter = project
+  .in(file("sdk/spring-boot-starter"))
+  .dependsOn(sdkSpring)
+  .enablePlugins(BuildInfoPlugin, PublishSonatype)
+  .settings(common)
+  .settings(
+    name := "kalix-spring-boot-starter",
+    crossPaths := false,
+    Compile / javacOptions ++= Seq("--release", "17"),
+    Compile / scalacOptions ++= Seq("-release", "17"),
+    buildInfoKeys := Seq[BuildInfoKey](
+      name,
+      version,
+      "protocolMajorVersion" -> Kalix.ProtocolVersionMajor,
+      "protocolMinorVersion" -> Kalix.ProtocolVersionMinor,
+      "scalaVersion" -> scalaVersion.value),
+    buildInfoPackage := "kalix.springboot",
```

I'm not too well-versed in sbt.. so mostly curious.. doesn't this need to be updated to?

```suggestion
    buildInfoPackage := "kalix.spring.boot",
```

---

Oh, good catch! 

And thanks for `feat!` suggestion.